src/optimize/base_optimizer.js

```diff
@@ -270,7 +270,6 @@ export default class BaseOptimizer {
               name: 'commons',
               chunks: (chunk) =>
                 chunk.canBeInitial() && chunk.name !== 'light_theme' && chunk.name !== 'dark_theme',
-              minChunks: 2,
```

Comment:
After migration "timeloin" `commons.styles.css` is no longer loaded.  Because of this bootstrap.js went to 'exception' path and I got an error related to redirect from old URL to new, also font_awesome styles didn't load. I deleted `minСhunks ` property so that `commons.styles` can be loaded.
![Capture](https://user-images.githubusercontent.com/16915480/85385357-e0af1780-b54a-11ea-8b82-1b99f3c0490f.PNG)

Reply:
Looks like this is outdated now (sorry, was on vacation last week)